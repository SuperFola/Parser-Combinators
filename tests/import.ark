(import a)
(import a.b)
(
import
foo.bar.egg
)
(import foo:*)
(import foo.bar:*)
(import foo.bar.egg:*)
(import foo :a)
(import foo.bar
:a
:b
)