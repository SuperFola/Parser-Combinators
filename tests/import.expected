( Keyword:import ( String:a ) ( ) )
( Keyword:import ( String:a String:b ) ( ) )
( Keyword:import ( String:foo String:bar String:egg ) ( ) )
( Keyword:import ( String:foo ) Symbol:* )
( Keyword:import ( String:foo String:bar ) Symbol:* )
( Keyword:import ( String:foo String:bar String:egg ) Symbol:* )
( Keyword:import ( String:foo ) ( Symbol:a ) )
( Keyword:import ( String:foo String:bar ) ( Symbol:a Symbol:b ) )